bin/run.sh

```diff
@@ -39,14 +41,18 @@ echo "${slug}: testing..."
 cd "${input_dir}" > /dev/null
 
 cp "${tests_file}" "${tests_file_original}"
+cp "${makefile}" "${makefile_original}"
 sed -i '/TEST_IGNORE\(\)/d' "${tests_file}"
+# tests need to be black and white for our python tooling
+sed -i 's/ -DUNITY_OUTPUT_COLOR//' "${makefile}"
```

Comment:
```suggestion
sed -i '/TEST_IGNORE\(\)/d' "${tests_file}"

cp "${makefile}" "${makefile_original}"
# tests need to be black and white for our python tooling
sed -i 's/ -DUNITY_OUTPUT_COLOR//' "${makefile}"
```

What do you think about reorganizing this so that the makefile and test file parts aren't interleaved?

Reply:
I just fixed the python regex so I'm not making any changes here.  I'm trying to do the minimal effort to make this work again.